src/psc_flatfoil_yz.cxx

```diff
@@ -128,70 +191,18 @@ using OutputParticles = PscConfig::OutputParticles;
 using Inject = typename InjectSelector<Mparticles, InjectFoil, Dim>::Inject;
 using Heating = typename HeatingSelector<Mparticles>::Heating;
 
-// ======================================================================
-// PscFlatfoilParams
-
-struct PscFlatfoilParams
-{
-  double BB;
-  double Zi;
-  double mass_ratio;
-  double lambda0;
-  double target_n;  // target density
-  double target_Te; // target electron temperature
-  double target_Ti; // target ion_temperatore
-  double target_Te_heat;
-  double target_Ti_heat;
-
-  double background_n;
-  double background_Te;
-  double background_Ti;
-
-  int inject_interval;
-
-  int heating_begin;
-  int heating_end;
-  int heating_interval;
-
-  // The following parameters are calculated from the above / and other
-  // information
-
-  double d_i;
-};
-
-// ======================================================================
-// Global parameters
-//
-// I'm not a big fan of global parameters, but they're only for
-// this particular case and they help make things simpler.
-
-// An "anonymous namespace" makes these variables visible in this source file
-// only
-namespace
-{
-
-// Parameters specific to this case. They don't really need to be collected in a
-// struct, but maybe it's nice that they are
-
-PscFlatfoilParams g;
 
-std::string read_checkpoint_filename;
-
-// This is a set of generic PSC params (see include/psc.hxx),
-// like number of steps to run, etc, which also should be set by the case
-PscParams psc_params;
 
-} // namespace
 
 // ======================================================================
 // setupParameters
 
 void setupParameters()
 {
   // -- set some generic PSC parameters
-  psc_params.nmax = 10000001; // 5001;
+  psc_params.nmax = 50;//10000001; // 5001;
   psc_params.cfl = 0.75;
-  psc_params.write_checkpoint_every_step = 1000;
+  psc_params.write_checkpoint_every_step = 50;
```

Comment:
Overall this looks good so far, but it also has a bunch of unrelated (to the HE electrons) changes. I think we can best clean that up in the end, though.